src/player.h

```diff
@@ -47,6 +47,9 @@ typedef char PlayerFlags[ PLAYER_FLAGS_MAX ];
 /* comfort flags. */
 #define player_isTut()     player_isFlag(PLAYER_TUTORIAL)
 
+/* dt_default grabber helper */
+#define player_dt_default() (player.p != NULL && player.p->ship != NULL ? player.p->ship->dt_default : 1)
```

Comment:
I don't know if it really matters in this case, but I'd probably be inclined to use a function instead of a macro when possible. A modern C compiler can inline the function to have the same effect.

Reply:
OK, done. It's a function now.